src/kotlinSessions/SessionTwo.kt

```diff
@@ -5,33 +5,36 @@ fun main() {
     //---------------------------------Loops----------------------------------
 
     // a loop is a block which is designed to be of a repeated execution
-    // so instead of saying
-    // println(x + 1)
+    // instead of saying
+    // var x: Int = 0
     // println(x + 1)
+    // println(x + 1) .....
     // we would commonly say
-    println("Test while loop")
+    println("Test for loop")
+    for (number in 0..2 step 1) {
+        println(number)
+    }
+
+    // the same code can be written using a while loop with operand inequalities operations (>=, <=, ==, !=)
+    println("Test equivalent while loop")
+    // var i = 0 and i <= 2 is the same as ordering a looper in the range 0..2
     var i: Int = 0
```

Comment:
If it is possible comment the function of loops in English like what the loop will be doing

Reply:
I will try.